tools/accuracy_checker/configs/handwritten-japanese-recognition-0001.yml

```diff
@@ -0,0 +1,40 @@
+models:
+  - name: handwritten-japanese-recognition-0001
+
+    launchers:
+      - framework: dlsdk
+        device: cpu
```

Comment:
please remove device, you can set it via `--target_device` command line argument